Binary files orig/gdtools/src/.Makevars.win.swp and patched/gdtools/src/.Makevars.win.swp differ
diff -Nru orig/gdtools/src/Makevars.win patched/gdtools/src/Makevars.win
--- orig/gdtools/src/Makevars.win	2023-10-15 12:38:54.000000000 +0200
+++ patched/gdtools/src/Makevars.win	2025-03-26 14:34:30.778722700 +0100
@@ -1,15 +1,12 @@
-RWINLIB = ../windows/cairo
+ifeq (,$(shell pkg-config --version 2>/dev/null))
+  PKG_CPPFLAGS = -I$(R_TOOLS_SOFT)/include/cairo \
+	         -I$(R_TOOLS_SOFT)/include/freetype2 \
+                 -DCAIRO_WIN32_STATIC_BUILD
+  PKG_LIBS = -lcairo -lfontconfig -lintl -liconv -lexpat -lfreetype -lharfbuzz \
+             -lpixman-1 -lpng -lbz2 -lz -lrpcrt4 -lgdi32 -lmsimg32
+else
+  PKG_CPPFLAGS = $(shell pkg-config --cflags cairo) 
+  PKG_LIBS = $(shell pkg-config --libs cairo)
+endif
 
-PKG_CPPFLAGS = -I$(RWINLIB)/include/cairo -I$(RWINLIB)/include \
-	-I$(RWINLIB)/include/freetype2 -I../inst/include -DCAIRO_WIN32_STATIC_BUILD
-
-PKG_LIBS = -L$(RWINLIB)/lib$(R_ARCH) -L$(RWINLIB)/lib -lcairo -lfreetype -lpng -lpixman-1 \
-  -lharfbuzz -lbz2 -lz -liconv -lgdi32 -lrpcrt4
-
-all: clean winlibs
-
-winlibs:
-	"${R_HOME}/bin${R_ARCH_BIN}/Rscript.exe" "../tools/winlibs.R"
-
-clean:
-	rm -f $(SHLIB) $(OBJECTS)
+PKG_CPPFLAGS += -I../inst/include
